src/datetime.rs

```diff
@@ -288,3 +289,87 @@ mod jiff_0_1 {
         }
     }
 }
+
+#[cfg(feature = "jiff_0_2")]
+mod jiff_0_2 {
+    use jiff::Span;
```

Comment:
This seems incorrect, right? It's the same in the jiff 0.1 module

Reply:
In the following line 296, we `use jiff_0_2 as jiff`. This alias is then used here.

For jiff 0.1, we do the same, just with `use jiff_0_1 as jiff`. It saves a bit of typing, but can be expanded, if you prefer.